server/lib/util.py

```diff
@@ -348,7 +348,6 @@ def get_nl_chart_titles():
 # Returns a set of SVs that should not have Per-capita.
 # TODO: Eventually read this from KG.
 def get_nl_no_percapita_vars():
-  # NOTE: This is a checked-in version of https://shorturl.at/afpMY
```

Comment:
In its place, could you please link to instructions on regenerating this that you used?